includes/admin/reporting/export/export-actions.php

```diff
@@ -194,47 +177,24 @@ function edd_include_file_downloads_batch_processer( $class ) {
 }
 
 /**
- * Register the API requests batch exporter
- *
- * @since  2.7
- */
-function edd_register_api_requests_batch_export() {
-	add_action( 'edd_batch_export_class_include', 'edd_include_api_requests_batch_processer', 10, 1 );
-}
-add_action( 'edd_register_batch_exporter', 'edd_register_api_requests_batch_export', 10 );
-
-/**
- * Loads the API requests batch process if needed
- *
- * @since  2.7
- * @param  string $class The class being requested to run for the batch export
- * @return void
- */
-function edd_include_api_requests_batch_processer( $class ) {
```

Comment:
Looks like this was unintentionally removed.

Exporting API requests 500 errors now.